src/routes/add-invoice.tsx

```diff
@@ -112,7 +135,7 @@ export default function AddInvoice() {
               <Button
                 color="secondary"
                 variant="contained"
-                onClick={handleCancel}
+                onClick={() => navigate('/')}
```

Comment:
It should be a link instead. This is not accessible and doesn't allow to use link native features (like click 3rd mouse button to open in new tab etc)